src/kartoffel/kartoffel.intergation.ts

```diff
@@ -112,41 +111,26 @@ export class Kartoffel {
      * @param dest? - optional param that identify the external destination, if not mentioned look in non-external network
      */
     public async searchByName(partialName: string, dest?: string): Promise<IUser[]> {
-        let res: AxiosResponse;
-        try {
-            const query: string =
-                dest && dest === (EXTERNAL_DESTS.CTS as any as string) ? kartoffelCTSQuerySearch : kartoffelQuery;
-            res = await this.instance.get(query, { params: { fullname: partialName } });
-        } catch (err) {
-            throw new ApplicationError(`Unknown Error: ${err} `);
-        }
-        const users: IKartoffelUser[] = res.data;
-        const usersWithRoles = users.filter(user => user?.hierarchy && Array.isArray(user.hierarchy));
-        const generalUsers: IUser[] = usersWithRoles.map((user: IKartoffelUser) => {
-          if (dest && dest === (EXTERNAL_DESTS.CTS as any as string)) {
-            // Get the id that match to cts datasource and replace the return id to cts id
-            const userMatch: IDomainUser[] = user.domainUsers.filter((domainUser) => {
-              return ctsDatasource === domainUser.dataSource;
-            });
-            user.id = userMatch[0].uniqueID ? userMatch[0].uniqueID : user.id;
-          }
-
-          return this.setUser(user);
+    let res: AxiosResponse;
+    try {
+        res = await this.instance.get(`/search`, { params: { fullName: partialName, expanded: true } });
```

Comment:
add 'digitalIdentity.source' in case of CTS dest

Reply:
Added.
If the dest is not CTS then the kartoffelQuery is: 
"/search?digitalIdentity.source=nonExternals"
But from the tests I did this request returns an empty array.
Why can't it be only "/search" when its not CTS?

Reply:
Done.